capstone-frontend/src/pages/BorderPage.jsx

```diff
@@ -44,62 +43,41 @@ function BorderPage() {
       .catch((error) => console.error(error));
   }
 
-  function createCommentDisplays(specificBorderPosts) {
-    allRelevantComments = [];
-    specificBorderPosts.forEach((element) => {
-      console.log(element);
-      // if (parseInt(element.borderNum) == borderIndex) {
-      //   allRelevantComments.push(element.userInput);
-      // }
-    });
-    return specificBorderPosts;
-  }
   const { data, error, isLoading } = useSWR(
     "http://localhost:5000/borderdata",
     fetcher
   );
   const crossingData = data;
 
-  useEffect(() => {}, [userComments]);
-
-  let position;
-  // let latitude = position.lat;
-  // let longitude = position.lng;
-  // position = { lat: latitude, lng: longitude };
   const fetchCoordinates = async (address) => {
     const apiKey = "AIzaSyDnk1NQgt08aY9-4tS0ZcG9WvzJc7hsuWE";
-    const url = `https://maps.googleapis.com/maps/api/geocode/json?address=${encodeURIComponent(address)}&key=${apiKey}`;
-    const response = await fetch(url)
+    const url = `https://maps.googleapis.com/maps/api/geocode/json?address=${encodeURIComponent(
```

Comment:
If you're having problems with this API, you can try to have a DB (or JSON file) on the backend which stores the coordinates you want so you don't have to deal with bad API.

Then you can send that down as part of the `borderdata` request and read it out on the front end. You can still use `Map` to render that.

Reply:
Discussed this option with Rahul & Kena; will look back at this as the core MVP is working. Most locations I am able to retrieve appropriate coordinates for; it is an error on the part of the API as a regular Google search works as expected.